packages/valory/customs/prediction_request/prediction_request.py

```diff
@@ -862,8 +863,14 @@ def fetch_additional_information(
         json_data = {"queries": [user_prompt]}
 
     if not source_links:
+        # limit the number of queries
+        if len(json_data["queries"]) > DEFAULT_NUM_QUERIES:
+            json_data["queries"] = json_data["queries"][:DEFAULT_NUM_QUERIES]
+        # remove empty queries, including ""
+        queries = json_data["queries"]
+        queries = [query for query in queries if query.strip() != ""]
```

Comment:
```suggestion
        # remove empty queries, including ""
        queries = json_data["queries"]
        queries = [query for query in queries if query.strip() != ""]
        # limit the number of queries
        if len(queries) > DEFAULT_NUM_QUERIES:
            queries = queries[:DEFAULT_NUM_QUERIES]
```
Reordered to make sure we don't skip valid queries in favor of invalid ones.
example queries: `["", "", "", "valid query", "valid query"]`

Reply:
241cfbf8bc64dd40b7a57373b3445ad3ba95603e